Docs/Configuration.tex

```diff
@@ -6824,6 +6802,81 @@ \subsubsection{Additional information}
 therefore \texttt{efibootmgr} rather than \texttt{bootctl} must be used for any low-level Linux command line interaction
 with the boot menu.
 
+\subsection{Other Boot Entry Protocol drivers}
+
+In addition to the \hyperref[uefilinux]{OpenLinuxBoot} plugin, the following \texttt{OC\_BOOT\_ENTRY\_PROTOCOL}
+plugins are made available to add optional, configurable boot entries to the OpenCore boot picker.
+
+\subsubsection{ResetNvramEntry}\label{uefiresetnvram}
+
+Adds a menu entry which resets NVRAM and immediately restarts. The \texttt{AllowNvramReset} option
+to allow resetting NVRAM using \texttt{CMD+OPT+P+R} is separate, and they can be enabled independently.
+
+See the warning about resetting NVRAM on Lenovo laptops given at the \texttt{AllowNvramReset} setting.
+
+The following configuration options may be specified in the \texttt{Arguments} section for this driver:
+
+\begin{itemize}
+  \item \texttt{-{}-preserve-boot} - Boolean flag, enabled if present. \medskip
+
+  If enabled, BIOS boot entries are not cleared during NVRAM reset. This option should
+  be used with caution, as some boot problems can be fixed by clearing these entries. \medskip
+
+  \item \texttt{-{}-native} - Boolean flag, enabled if present. \medskip
```

Comment:
Hmm. At first I thought this difference in naming comes from different background assumptions - I got into OpenCore pretty late, running it in on real unsupported Macs, while you built it, obviously got into it very early(!), and running on non-Apple hardware. So I think it's 'native', because I'm thinking about Aapl hardware... But then again, large parts of OpenCore emulate parts of Apple early boot which aren't in non-Apple hardware (&/or replace/modify similar things which are there). So it still seems pretty reasonable to say that this feature triggers the native version of something which OpenCore otherwise 'emulates' (does with it's own version). That said, and if you're still not convinced, ofc I'm happy to change the naming!

Reply:
I can't point at specific examples right now, but I seem to get the impression that OC generally uses `apple` for what you describe and that this will fit better with existing instances. Also, `native` could be misunderstood by a casual user as being a "native" OpenCore item/implementation as compared to a third party item/implementation.

Reply:
Yeah, fair point.